toga/interface/dialogs.py

```diff
@@ -0,0 +1,22 @@
+def info_dialog(window, title, message):
```

Comment:
The renaming of these APIs conflicts with toga-demo, and the API expected by [toga.Window](https://github.com/pybee/toga-core/blob/master/toga/interface/window.py#L137). Since these are modal dialogs, the expected usage is via `window.info_dialog()`.